packages/types/src/types/ProvidersInventory.ts

```diff
@@ -9,140 +14,3 @@ export interface InventoryByType {
   ovirt?: OVirtInventory[] | null;
   vsphere?: VSphereInventory[] | null;
 }
-
-/**
- * General provider inventory
- */
-export type ProviderInventory =
```

Comment:
the file is named by the main export, in the case `ProviderInventory` if you use this file to export different types, the file name should also changed